azurerm/internal/services/network/network_connection_monitor_resource_test.go

```diff
@@ -234,6 +234,35 @@ func testAccNetworkConnectionMonitor_icmpConfiguration(t *testing.T) {
 	})
 }
 
+func testAccNetworkConnectionMonitor_endpointType(t *testing.T) {
```

Comment:
Can we rename this function to indicate it's covering a deprecated property?

```suggestion
func testAccNetworkConnectionMonitor_endpointDeprecated(t *testing.T) {
```

Reply:
Updated